.gitignore

```diff
@@ -1,4 +1,5 @@
 .idea/
 assets/x2t/x2t
 results/
-spec/test_data/tmp
\ No newline at end of file
+spec/test_data/tmp
+.bundle/
```

Comment:
Add newline in end of file so there is no such symbol

![image](https://user-images.githubusercontent.com/668524/129549462-aa2e0482-32ea-4c22-924b-e55be80739ba.png)